dali/dfu/dfurun.cpp

```diff
@@ -1117,6 +1117,26 @@ class CDFUengine: public CInterface, implements IDFUengine
             opttree->setProp("@encryptKey",encryptkey);
             opttree->setProp("@decryptKey",decryptkey);
         }
+
+        // if maxConnection is not passed as a user option, check for a default defined in the config/environment.
+        if (!opttree->hasProp("@maxConnections"))
+        {
+            int configDefault = -1;
+            if (isContainerized())
```

Comment:
This is the one situation where you need to use #ifdef _CONTAINERIZED rather than isContainerized().  

The containerized version will not define getEnvironmentFactory(), so you will get link errors/warnings if symbol resolution is enforced at link time.

Reply:
I see. Changed.